volatility3/framework/symbols/windows/extensions/__init__.py

```diff
@@ -762,7 +765,9 @@ def to_list(self,
             obj_offset = link.vol.offset - relative_offset
 
             try:
```

Comment:
I think it used to throw an exception, which if instead it just returns a `bool` means that this exception seems redundant now?  The one above may cover the dereference, but this is definitely just for the `is_valid` call.

Reply:
Some layers throw and exception, some don't.
I can remove the try catch if you want

Reply:
Well, the point of `is_valid` is that you don't need to do the `try/except`, so I'd prefer to find anything where `is_valid` throws an exception, and instead get it to catch and throw false instead.